openhcl/openhcl_boot/src/main.rs

```diff
@@ -508,6 +511,18 @@ const fn zeroed<T: FromZeroes>() -> T {
     unsafe { core::mem::MaybeUninit::<T>::zeroed().assume_init() }
 }
 
+/// Get the tsc ref time for hw isolated vms in 100ns
+#[allow(unused_variables)]
+fn get_hw_tsc_time(isolation: IsolationType) -> u64 {
```

Comment:
@smalis-msft do you know what it would take to do the same on SNP?

Reply:
Might be a better question for @sluck-msft? I haven't dug into the SNP docs nearly deep enough to know off the top of my head

Reply:
@adi-bhandari or @daaltobe maybe?